BUILDING.md

```diff
@@ -1,244 +1,102 @@
-  { license-url = "file:///path/to/spdx-id-license.txt", path = "spdx-id-license.txt" },
-  # This file is fetched from an https endpoint, and will be saved as `path`
-  { license-url = "https://localhost/spdx-id-license-v2.txt", path = "spdx-id-license-2.txt" }
-]
+After the kit has been built locally the kit can be published to the provided vendor in Infra.toml. This operation can also be handled by the makefile.
 ```
-
-#### NVIDIA variants
-
-If you want to build any of the NVIDIA variants, you can follow these steps to prepare a `Licenses.toml` file using the [License for customer use of NVIDIA software](https://www.nvidia.com/en-us/drivers/nvidia-license/):
-
-1. Create a `Licenses.toml` file in your Bottlerocket root directory, with the following content:
-
-```toml
-[nvidia]
-spdx-id = "LicensesRef-NVIDIA-Customer-Use"
-licenses = [
-  { path = "LICENSE", license-url = "https://www.nvidia.com/en-us/drivers/nvidia-license/" }
-]
+make publish VENDOR=<vendor-name>
 ```
+At this point, there should be a core-kit image in ECR which can be consumed when building a variant to test and validate.
+## Building a Variant with the published kit image
+This section will cover building a variant to test a build of the core kit as done above. Please note this section does not cover the complete complexity of testing a change to the bottlerocket OS. For this see the [BUILDING](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md) section in the [bottlerocket](https://github.com/bottlerocket-os/bottlerocket/) repository.
 
-2. Fetch the licenses with this command:
-
-```shell
-cargo make -e BUILDSYS_UPSTREAM_LICENSE_FETCH=true fetch-licenses
-```
-
-3. Build your image, setting the `BUILDSYS_UPSTREAM_SOURCE_FALLBACK` flag to `true`, if you haven't cached the driver's sources:
-
-```shell
-K8S_VERSION=1.24
-cargo make \
-  -e BUILDSYS_VARIANT=aws-k8s-${K8S_VERSION}-nvidia \
-  -e BUILDSYS_UPSTREAM_SOURCE_FALLBACK="true"
-```
-
-### Register an AMI
-
-To use the image in Amazon EC2, we need to register the image as an AMI.
-
-To do this, you'll need to have your AWS account credentials setup on your system.
-There are lots of ways to do this; one method is using [the `aws` CLI](https://aws.amazon.com/cli/) via its `configure` command with your user's access and secret keys.
-If you're using an EC2 instance, the [EC2 instance's IAM role](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) will be used automatically if available.
-
-For a simple start, pick an [EC2 region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions), then run:
-
-```shell
-cargo make -e PUBLISH_REGIONS=your-region-here ami
+### Setup Twoliter.toml for testing
+To consume a private copy of the bottlerocket core kit with your changes built into it, you need to define the vendor that points to your container registry in `Twoliter.toml` and adjust the core kit dependency:
```

Comment:
nit: I didn't look for other occurrences but please take another editing pass and be sure to use "Bottlerocket" consistently:

```suggestion
To consume a private copy of the Bottlerocket core kit with your changes built into it, you need to define the vendor that points to your container registry in `Twoliter.toml` and adjust the core kit dependency:
```